xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -3772,6 +3772,7 @@ platform.dashboard.user.displayOnMainPage=Display my dashboard on the wiki home
 platform.dashboard.wiki=Dashboard
 platform.dashboard.wiki.pages=Pages
 platform.dashboard.wiki.tagcloud=Tags
+platform.dashboard.wiki.tagsgadget=Tags for {0}
```

Comment:
I didn't want to use the word 'space' in a translation key (actually that was my first idea, too). But I'll change it.